packages/delegation-process/src/services/delegationService.ts

```diff
@@ -21,17 +21,29 @@ import {
 import { ReadModelService } from "./readModelService.js";
 import { assertRequesterIsDelegateOrDelegator } from "./validators.js";
 
-export const retrieveDelegationById = async (
+const retrieveDelegationById = async (
```

Comment:
Nice! By removing the export, in the other services we won't risk to use this instead of the other one